prechecks.yml

```diff
@@ -6,39 +6,60 @@
 - hosts: localhost
   connection: local
   tasks:
-    # For dig lookup module[1] to work (dig lookup is used later in this
-    # playbook), you need to have python dns module[2] installed[3] on ansible
-    # control machine (the local machine where you run ansible).
-    # [1] https://docs.ansible.com/ansible/latest/playbooks_lookups.html#the-dns-lookup-dig
-    # [2] http://www.dnspython.org/
-    # [3] Fedora package: python2-dns, RHEL 7 package: python-dns
-    # This check is here to provide more clear error message when the module
-    # is missing.
-    - name: Check that python dns is installed on ansible control machine
+
+    - name: Check if python dns is installed on ansible control machine
```

Comment:
Do we really need python-dns? I think earlier we were talking about droping the dig lookups like --.https://github.com/Tendrl/tendrl-ansible/blob/master/site.yml.sample#L46 <-- and leave it for user configuration

Reply:
You are right that dig lookups are going to be removed from site.yml, but there is one task which uses it in prechecks playbook (Check that fqdn of the host is available via dns) - which I can make optional or remove it altogether based on upcoming instructions how to setup host names when not using dns.

Reply:
The reason I would keep the dns check here is that we are not testing it right now (we use dns setup in our test environment) and we require fqdn on Tendrl side for some internal reasons, so just removing the check without a guide how to set up /etc/hosts files could create problems.

If you already have a draft of guide how to setup hosts file on machines for Tendrl, add it to the wiki and link it there, so I can change the check based on it in this pull request. Otherwise I suggest to keep it as it is and do the change later in a separate pull request.